tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "downlevelIteration": true
```

Comment:
I would avoid adapting the `tsconfig`. When iterating over sets etc where you would need this flag you can always use `Array.from(...)` instead.